package.json

```diff
@@ -1,42 +1,59 @@
 {
-  "name": "@digitalbazaar/isomorphic-js-lib-template",
-  "version": "0.0.1",
-  "description": "",
-  "homepage": "https://github.com/digitalbazaar/isomorphic-js-lib-template",
+  "name": "bedrock-web-local-vc-store",
+  "version": "1.0.0",
```

Comment:
I think this might need to be something less than 1.0.0 for tools to work?  More like 1.0.0-0 or 0.0.1.